.circleci/config.yml

```diff
@@ -123,26 +130,93 @@ jobs:
           key: go-mod-helm-v4-{{ checksum "go.sum" }}
           paths:
             - "/go/pkg/mod"
+  helm-build:
+    docker:
+      - image: gcr.io/public-builds/chart-releaser:v1.2.1
+    steps:
+      - checkout
+      - run:
+          name: install yq
+          command: |
+            wget https://github.com/mikefarah/yq/releases/download/v4.23.1/yq_linux_amd64
+            mv yq_linux_amd64 /usr/local/bin/yq
+            chmod +x /usr/local/bin/yq
+      - run:
+          name: install helm
+          command: |
+            wget -q https://get.helm.sh/helm-v3.5.2-linux-amd64.tar.gz -O - | tar -xzO linux-amd64/helm > /usr/local/bin/helm && chmod +x /usr/local/bin/helm
+      - run: 
+          name: make cr release dirs
+          command: |
+            mkdir -p cr-release-packages/official
+            mkdir -p cr-release-packages/preview
+      # - run: helm package -d .cr-release-packages/ --version ${CIRCLE_TAG:1} --app-version ${CIRCLE_TAG:1} etc/helm/pachyderm # cr package did not work with embedded chart
+      # TODO: remove and replace only for testing.
+      - run:
```

Comment:
This double build is a bit awkward. I think it would be cleaner to push to a different staging helm repo which is not indexed by artifacthub. We'll likely need to use the `yq` code in other places though (ie. when doing a custom version / alpha)

Reply:
yea i agree its a bit awkward. but introducing a new repo for the helm chart just comes with a lot more to change atm. the cost of repacking files to archive is less than a minute so thought it could be a temporary evil. ideal id like to replace all this with something like chart museum.